packages/performance-tests/test/rawInserts.ts

```diff
@@ -0,0 +1,175 @@
+
+  Logger.logInfo(loggerCategory, "starting 150k entity inserts");
+
+  const [insertsTimer] = timed(() => {
+    const physModelId = PhysicalModel.insert(sourceDb, IModelDb.rootSubjectId, "physical model");
+    const categoryId = SpatialCategory.insert(sourceDb, IModelDb.dictionaryId, "spatial category", {});
+
+    // 100,000 elements, 50,000  relationships
+    for (let i = 0; i < ELEM_COUNT / 2; ++i) {
+      const [id1, id2] = [0, 1].map((n) => new PhysicalObject({
+        classFullName: PhysicalObject.classFullName,
+        category: categoryId,
+        geom: IModelTransformerTestUtils.createBox(Point3d.create(i, i, i)),
+        placement: {
+          origin: Point3d.create(i, i, i),
+          angles: YawPitchRollAngles.createDegrees(i, i, i),
+        },
+        model: physModelId,
+        code: new Code({ spec: IModelDb.rootSubjectId, scope: IModelDb.rootSubjectId, value: `${2*i + n}`}),
+        userLabel: `${2*i + n}`,
+      }, sourceDb).insert());
+
+      const rel = new ElementGroupsMembers({
+        classFullName: ElementGroupsMembers.classFullName,
+        sourceId: id1,
+        targetId: id2,
+        memberPriority: i,
+      }, sourceDb);
+
+      rel.insert();
+    }
+  });
+
+  reporter.addEntry(
+    "populate by insert",
+    iModelName,
+    "time elapsed (seconds)",
+    insertsTimer?.elapsedSeconds ?? -1,
+    {
+      elementCount: IModelTransformerTestUtils.count(sourceDb, "Bis.ElementGroupsMembers"),
+      relationshipCount: IModelTransformerTestUtils.count(sourceDb, "Bis.Element"),
+    }
+  );
+
+  sourceDb.saveChanges();
+
+  Logger.logInfo(loggerCategory, "Done. Starting changeset application of same content");
+
+  const changeset1 = createChangeset(sourceDb);
+  const changesetDbPath = initOutputFile(`RawInsertsApply.bim`, outputDir);
+  if (fs.existsSync(changesetDbPath))
+    fs.unlinkSync(changesetDbPath);
+  const changesetDb = StandaloneDb.createEmpty(changesetDbPath, { rootSubject: { name: "RawInsertsApply" }});
+
+  const [applyChangeSetTimer] = timed(() => {
+    changesetDb.nativeDb.applyChangeset(changeset1);
+  });
+
+  reporter.addEntry(
+    "populate by applying changeset",
+    iModelName,
+    "time elapsed (seconds)",
+    applyChangeSetTimer?.elapsedSeconds ?? -1,
+    {
+      elementCount: IModelTransformerTestUtils.count(changesetDb, "Bis.ElementGroupsMembers"),
+      relationshipCount: IModelTransformerTestUtils.count(changesetDb, "Bis.Element"),
+    }
+  );
+
+  Logger.logInfo(loggerCategory, "Done. Starting with-provenance transformation of same content");
+
+  const targetPath = initOutputFile(`RawInserts-Target.bim`, outputDir);
+  const targetDb = SnapshotDb.createEmpty(targetPath, { rootSubject: { name: "RawInsertsTarget" }});
+  const transformerWithProv = new IModelTransformer(sourceDb, targetDb, { noProvenance: false });
+
+  const [transformWithProvTimer] = await timed(async () => {
+    await transformerWithProv.processAll();
+  });
```

Comment:
You don't log the timer like you did for identity transform. Might be useful, but not really a big deal.

Reply:
I don't think we need it for these 4 tests. Later we can add a wrapper around `reporter.addEntry` that logs the times locally and writes them to an in-progress file rather than report.csv only at the end